charts/edge-site/values.yaml

```diff
@@ -19,7 +19,16 @@ globals:
     # The bucket name where the deployment will send the requested files
     bucketName:
 
+  # If using an S3-compatible service as the controller store provider, configure its endpoint here.
+  s3CompatibleService:
+    # Url including scheme of the base API endpoint for this service. `http` and `https` schemes
+    # are supported.
+    url: ""
+    # If your service is configured with a region (eg. us-east-1), set it here.
+    region: ""
+
 edgeController:
+  storageProvider: "local"
   storageClaim: edge-controller-storage-claim
```

Comment:
If you're running with minio for storage, do you still need this storage claim?

Reply:
I made this conditional in `statefulset.yaml`.